pkgs/development/haskell-modules/configuration-darwin.nix

```diff
@@ -323,6 +323,20 @@ self: super: ({
   # Tests fail on macOS https://github.com/mrkkrp/zip/issues/112
   zip = dontCheck super.zip;
 
+  warp = super.warp.overrideAttrs (drv: {
+    __darwinAllowLocalNetworking = true;
```

Comment:
How did you hit this? Seems like building `warp` hasn't been a problem so far.

Reply:
Might be some combination of sandbox + firewall. I'll try to isolate the trigger

Reply:
This builds for me with `--option sandbox false`, does not build with `relaxed` or `true`.
Test suite just hangs after the `"range requests"` group log and there's no more output:
https://github.com/yesodweb/wai/blob/d50e1184600631a114e3d2ad119abdf0ef08834b/warp/test/ResponseSpec.hs#L106
I suspect this is the first case in the suite that actually performs a network request (apparently in `127.0.0.1` flavor)

Reply:
Do you hit this with just GHC 9.8? warp does seem to succeed with GHC 9.6 which is confusing to me.

Reply:
On further testing it looks like it only happens when both firewall AND sandboxing are active. 
Didn't try 9.8.